postgresql/installingpostgresqlonibmi.md

```diff
@@ -0,0 +1,167 @@
+# Install and configure Postgresql DB Server on IBM i 
+
+Install postgres yum packages from IBM i ACS Open Source Package Management
+```
+Package list:
+postgresql12 ***(This item may be the only one you need to select to include all packages)***
+postgresql12-contrib
+postgresql12-devel
+postgresql12-docs
+postgresql12-libecpg
+postgresql12-libogtypes
```

Comment:
Typo should be `postgresql12-libpgtypes`.

Reply:
Maybe we should just merge it and clean it up ourselves to give Richard back his time.

Reply:
I don't mind. Just seems that would be much faster :-)

Then I can pull your corrections.

Reply:
> Maybe we should just merge it and clean it up ourselves to give Richard back his time.

Sounds good